### PR TITLE
Fixed freezing of an app, when user enters news feed content.

### LINKED ISSUE #602: Freezing of an App when User enters less content.
Issue Description:
When the details with "post title" and "write your post" text fields are given content of news feed, it causes app to freeze and stop working.

Expected Behavior:
It should show some error message in toast widget.

Does this issue need immediate attention?
yes

Are you willing to work on this issue:
Yes

### PR DESCRIPTION
**Pull request by amul-dhungel**
What kind of change does this PR introduce?

Bug fix, the exception toast was not working in add post. So, I replaced the exception toast arguments with proper
validator. #602 

Did you add tests for your changes?
No
If relevant, did you update the documentation?
No
Summary
This PR solves the Issue #602. The app should not freeze, when the user enters the news feed content.

Snapshot
Screenshot_2021-04-04-14-20-16-815_com example talawa

Does this PR introduce a breaking change?
A moderate change.

Other information
Fixes #602

### CODE REVIEW

**Comment by Sagar2366** [comment]
@amul-dhungel please disable code auto formatter before adding commits to PR and send only changes required to fix this issue. Do not merge previous commits to your PR.

**Comment by amul-dhungel** [comment]
@Sagar2366  Yes sir got it

File: lib/main.dart [hunk 1]
```diff
@@ -19,9 +19,12 @@ import 'views/pages/organization/switch_org_page.dart';
 Preferences preferences = Preferences();
 String userID;
 Future<void> main() async {
-  WidgetsFlutterBinding.ensureInitialized(); //ensuring weather the app is being initialized or not
+  WidgetsFlutterBinding
```
Comment (Sagar2366) [inline lib/main.dart:22]: remove this change, not required to fix the issue.

File: lib/main.dart [hunk 2]
```diff
@@ -19,9 +19,12 @@ import 'views/pages/organization/switch_org_page.dart';
...
+      .ensureInitialized(); //ensuring weather the app is being initialized or not
   userID = await preferences.getUserId(); //getting user id
-  SystemChrome.setPreferredOrientations([DeviceOrientation.portraitUp])  //setting the orientation according to the screen it is running on
+  SystemChrome.setPreferredOrientations([
```
Comment (Sagar2366) [inline lib/main.dart:25]: remove this change, not required to fix the issue.

File: lib/main.dart [hunk 3]
```diff
@@ -73,7 +77,9 @@ class MyApp extends StatelessWidget {
           WidgetBuilder builder = routes[settings.name];
           return MaterialPageRoute(builder: (ctx) => builder(ctx));
         },
-        home: userID == null ? UrlPage() : HomePage(), //checking weather the user is logged in or not
+        home: userID == null
```
Comment (Sagar2366) [inline lib/main.dart:80]: remove this change, not required to fix the issue.

**Comment by Sagar2366** [comment]
@amul-dhungel you still have not removed auto code formatter code.

**Comment by Sagar2366** [comment]
> @amul-dhungel you still have not removed auto code formatter code.

@amul-dhungel let us know if you don't want to continue fixing this issue.

**Pull request closed without merging**